### ISSUE TITLE
can't download ie6 (404 not found)

### ISSUE THREAD

**Issue by GoogleCodeExporter** [noahocker/winetricks#118]
```
What steps will reproduce the problem?
1. sh winetricks ie6

What is the expected output? What do you see instead?
expected: download and install ie6
output: 
HTTP request sent, awaiting response... 404 Not Found
2011-08-24 11:44:02 ERROR 404: Not Found.

------------------------------------------------------
Downloading http://browsers-us.mirrors.zensoft.net/ie/win32/6.0/ie60.exe failed
------------------------------------------------------

What version of the product are you using? On what operating system?
20110629 on Arch Linux

Please provide any additional information below.


```

Original issue reported on code.google.com by `czesud...@gmail.com` on 24 Aug 2011 at 9:50

**Comment by GoogleCodeExporter** [comment]
```
Fixed in svn.   
```

Original comment by `daniel.r...@gmail.com` on 24 Aug 2011 at 2:43
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
I just ran into the same problem. I got around it like this:

I downloaded the file from: 
http://www.oldversion.com/download-Internet-Explorer-6.0.html (I had to 
manually download the file, it uses a post method to get the file and I just 
don't really know a way around that).

The file you download is msie60.exe. Now you need to change line 7350 in 
winetricks

from 
w_download http://browsers-us.mirrors.zensoft.net/ie/win32/6.0/ie60.exe 
8e483db28ff01a7cabd39147ab6c59753ea1f533

to
w_download http://127.0.01/msie60.exe 8e483db28ff01a7cabd39147ab6c59753ea1f533


Assuming you have an apache server running. Otherwise, you need to host the 
file elsewhere and change the line to reflect that location.
```

Original comment by `bth5...@gmail.com` on 5 Sep 2011 at 5:53

**Comment by GoogleCodeExporter** [comment]
```
Or you could just use the updated winetricks in svn.
```

Original comment by `daniel.r...@gmail.com` on 5 Sep 2011 at 6:16

**Comment by GoogleCodeExporter** [comment]
```
Issue 124 has been merged into this issue.
```

Original comment by `austinenglish@gmail.com` on 26 Oct 2011 at 7:09

**Comment by GoogleCodeExporter** [comment]
```
Or you can download the file from the website above, and then move the file to 
~/.cache/winetricks/ie6 and run winetricks ie6.
```

Original comment by `markx...@gmail.com` on 27 Oct 2011 at 7:03

**Comment by GoogleCodeExporter** [comment]
```
"""Or you can download the file from the website above, and then move the file 
to ~/.cache/winetricks/ie6 and run winetricks ie6."""  

Worked but but go to the "~/.cache/winetricks/ie6" directory and make sure to 
rename the msie60.exe to ie60.exe.  If that's not obvious. :)

```

Original comment by `mcd...@gmail.com` on 3 Nov 2011 at 3:56

**Comment by GoogleCodeExporter** [comment]
```
Thanks for the notes guys.  I couldn't have done this without you.
```

Original comment by `kit.chue...@gmail.com` on 12 Dec 2011 at 5:47

**Comment by GoogleCodeExporter** [comment]
```
Why did you need the workaround?  Isn't winetricks-20111115 working for you?
```

Original comment by `daniel.r...@gmail.com` on 12 Dec 2011 at 12:39

**Comment by GoogleCodeExporter** [comment]
```
I had the same issue, it seems that zensoft mirror is missing the file. 
winetricks itself works fine, but the link for ie6 doesn't seem to work anymore.
```

Original comment by `sligo...@gmail.com` on 7 Feb 2012 at 10:37

**Comment by GoogleCodeExporter** [comment]
```
You're using an old version of winetricks.  This bug was fixed in r690 on 
2001-08-20.
```

Original comment by `daniel.r...@gmail.com` on 7 Feb 2012 at 2:39

**Comment by GoogleCodeExporter** [comment]
```
The file you download is msie60.exe. Now you need to change line 7350 in 
winetricks

from 
w_download http://browsers-us.mirrors.zensoft.net/ie/win32/6.0/ie60.exe 
8e483db28ff01a7cabd39147ab6c59753ea1f533

to
w_download http://www.oldversion.com/common/uploaded_files/softwares/msie60.exe 
8e483db28ff01a7cabd39147ab6c59753ea1f533


Assuming you have an apache server running. Otherwise, you need to host the 
file elsewhere and change the line to reflect that location.
```

Original comment by `brem...@gmail.com` on 24 Feb 2012 at 1:17

**Comment by GoogleCodeExporter** [comment]
```
winetricks currently has the line 
    w_download http://download.oldapps.com/Internet_Explorer/ie60.exe 8e483db28ff01a7cabd39147ab6c59753ea1f533
which works.  Anyone with the old line has an old winetricks.

```

Original comment by `daniel.r...@gmail.com` on 24 Feb 2012 at 3:00

**Comment by GoogleCodeExporter** [comment]
```
i used host file to redirect http://browsers-us.mirrors.zensoft.net to 127.0.0.1
then hosted the file under 127.0.0.1/ie/win32/6.0/ie60.exe with apache2
```

Original comment by `jvance...@gmail.com` on 23 Mar 2012 at 11:08

**Comment by GoogleCodeExporter** [comment]
```
It would be easier to update to the latest winetricks, wouldn't it?

What does 'winetricks --version' say?
The current version says 20120308, and does not suffer from this problem.
```

Original comment by `daniel.r...@gmail.com` on 23 Mar 2012 at 11:14

**Comment by GoogleCodeExporter** [comment]
```
Using 20120308 here with wineskin. Getting this;

Executing w_do_call ie6
Executing load_ie6
Executing mkdir -p /Users/stephenbelanger/Library/Caches/winetricks/ie6
Downloading http://download.oldapps.com/Internet_Explorer/ie60.exe to 
/Users/stephenbelanger/Library/Caches/winetricks/ie6
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0 76.7M    0  194k    0     0   262k      0  0:04:59 --:--:--  0:04:59  295k
  1 76.7M    1  831k    0     0   324k      0  0:04:01  0:00:02  0:03:59  335k
  1 76.7M    1  831k    0     0   233k      0  0:05:36  0:00:03  0:05:33  239k
  1 76.7M    1  831k    0     0   182k      0  0:07:11  0:00:04  0:07:07  185k
  1 76.7M    1  831k    0     0   149k      0  0:08:45  0:00:05  0:08:40  151k
  1 76.7M    1  831k    0     0   126k      0  0:10:20  0:00:06  0:10:14  109k
  1 76.7M    1  831k    0     0   119k      0  0:10:57  0:00:06  0:10:51     0
  1 76.7M    1  831k    0     0   119k      0  0:10:57  0:00:06  0:10:51     0
curl: (18) transfer closed with 79620934 bytes remaining to read
------------------------------------------------------
Downloading http://download.oldapps.com/Internet_Explorer/ie60.exe failed
------------------------------------------------------
```

Original comment by `step...@opzi.com` on 3 Jul 2012 at 5:56
